crates/ruff_python_formatter/tests/snapshots/black_compatibility@simple_cases__attribute_access_on_number_literals.py.snap

```diff
@@ -34,59 +34,42 @@ y = 100(no)
 ```diff
 --- Black
 +++ Ruff
-@@ -1,19 +1,19 @@
- x = (123456789).bit_count()
- x = (123456).__abs__()
--x = (0.1).is_integer()
--x = (1.0).imag
--x = (1e1).imag
--x = (1e-1).real
-+x = (.1).is_integer()
-+x = (1.).imag
-+x = (1E+1).imag
-+x = (1E-1).real
- x = (123456789.123456789).hex()
--x = (123456789.123456789e123456789).real
--x = (123456789e123456789).conjugate()
--x = 123456789j.real
+@@ -8,10 +8,10 @@
+ x = (123456789.123456789e123456789).real
+ x = (123456789e123456789).conjugate()
+ x = 123456789j.real
 -x = 123456789.123456789j.__add__(0b1011.bit_length())
 -x = 0xB1ACC.conjugate()
 -x = 0b1011.conjugate()
 -x = 0o777.real
-+x = (123456789.123456789E123456789).real
-+x = (123456789E123456789).conjugate()
-+x = 123456789J.real
-+x = 123456789.123456789J.__add__((0b1011).bit_length())
-+x = (0XB1ACC).conjugate()
-+x = (0B1011).conjugate()
-+x = (0O777).real
++x = 123456789.123456789j.__add__((0b1011).bit_length())
++x = (0xB1ACC).conjugate()
```

Comment:
now that we have no more trailing dots in floats we can also fix the parentheses rules for them (maybe better in a follow-up PR so that the scope of this PR doesn't get too large)